src/main/java/org/reaktivity/nukleus/http2/internal/stream/Http2ServerFactory.java

```diff
@@ -1206,7 +1202,7 @@ private void cleanup()
 
             if (!streams.isEmpty())
             {
-                executor.schedule(100, TimeUnit.MILLISECONDS, routeId, replyId, CLEANUP_SIGNAL);
+                signaler.signalAt(Instant.now().plusMillis(100).toEpochMilli(), routeId, replyId, CLEANUP_SIGNAL);
```

Comment:
Please hoist `timeMillis` parameter value for readability rather than inlining with method call.